docs/tracking-javascript-guide.md

```diff
@@ -693,7 +693,65 @@ _paq.push(['trackPageView']);
 
 ## Multiple Piwik trackers
 
-It is possible to track a page using multiple Piwik trackers that point to the same or different Piwik servers. To improve page loading time, you can load `piwik.js` once. Each call to `Piwik.getTracker()` returns a unique Piwik Tracker object (instance) which can be configured.
+By default, the Piwik JavaScript Tracking code collects your analytics data into one Piwik server. The Piwik service URL is specified in your JavaScript Tracking code (for example: `var u="//piwik.example.org";`). In some cases, you may want to track your analytics data into more than just one Piwik server. 
```

Comment:
"... into more than just one Piwik server. " could be maybe (but even ok the way it is) "... into more than just one Piwik server or into multiple websites on the same Piwik server at once".

Also not sure if "server" is the correct word as there could be multiple servers in one Piwik installations and it may confuse some technical people to use that term